submissions/2020-07-28-OpenFF-Disaccharide-Optimization-Set/DAllpa1-2DAllpa1-OME.json

```diff
@@ -0,0 +1,326 @@
+{
+  "dataset_name": "mdgx-created data set",
+  "dataset_tagline": "A data set created by mdgx",
+  "dataset_type": "OptimizationDataSet",
+  "method": "d3bj b3lyp",
```

Comment:
This model won't be incorporated unless it is in a spec

Reply:
Actually this is my mistake, I believe this is how QCSubmit handles these fields